test/new-e2e/tests/sysprobe-functional/usmtest/test_tags.ps1

```diff
@@ -0,0 +1,130 @@
+
```

Comment:
Is there a specific reason this is in PS?

Reply:
The code has to execute on the test vm.
the `go` code executes on a controller box (which in CI is linux).
So, I needed something that's always present on the target machines, and can reliably generate the test activity and retrieve the test responses.